sample/EasyTables/CSharp/QueueToInItemToOutItem/run.csx

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using Microsoft.Azure.WebJobs.Host;
+using Newtonsoft.Json.Linq;
+
+public static void Run(string input, Item queriedItem, out Item newItem)
+{
+    newItem = new Item();    
+    newItem.Text = input;
+    queriedItem.Text += $"...{input}";
+}
+
+public class Item
```

Comment:
We need a way to share model types like this. As it stands now, it looks like each of your samples includes the same class. @fabiocav is there a pattern for sharing code already established?